src/Kros.MassTransit.AzureServiceBus/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -22,7 +22,7 @@ public static IServiceCollection AddMassTransitForAzure(
             this IServiceCollection services,
             IConfiguration configuration,
             Type consumerNamespaceAnchor,
-            Action<IMassTransitForAzureBuilder> busCfg = null)
+            Action<IMassTransitForAzureBuilder, IBusRegistrationContext> busCfg = null)
```

Comment:
Nedávaj to ako druhý parameter do existujúcej akcie. Takto ak niekto akciu používa, bude to pre neho breaking change a po aktualizácii si bude musieť meniť veci. Daj to ako nový parameter `Action<IBusRegistrationContext> contextCfg = null`.

Alebo je nejaký use case, kde potrebuješ obe veci naraz?